test/ServiceBus.Tests/SlowConsumingTests/EHSlowConsumingTests.cs

```diff
@@ -0,0 +1,178 @@
+        private const double flowControlThredhold = 0.6;
+        public static readonly EventHubStreamProviderSettings ProviderSettings =
+            new EventHubStreamProviderSettings(StreamProviderName);
+
+        private static readonly Lazy<EventHubSettings> EventHubConfig = new Lazy<EventHubSettings>(() =>
+            new EventHubSettings(
+                TestDefaultConfiguration.EventHubConnectionString,
+                EHConsumerGroup, EHPath));
+
+        private static readonly EventHubCheckpointerSettings CheckpointerSettings =
+            new EventHubCheckpointerSettings(TestDefaultConfiguration.DataConnectionString,
+                EHCheckpointTable, CheckpointNamespace, TimeSpan.FromSeconds(1));
+
+
+        private readonly Fixture fixture;
+
+        public class Fixture : BaseTestClusterFixture
+        {
+            protected override TestCluster CreateTestCluster()
+            {
+                var options = new TestClusterOptions(2);
+                ProviderSettings.SlowConsumingMonitorFlowControlCheckperiod = monitorCheckperiod;
+                ProviderSettings.SlowConsumingMonitorFlowControlThreshold = flowControlThredhold;
+                ProviderSettings.AveragingCachePressureMonitorFlowControlThreshold = null;
+                AdjustClusterConfiguration(options.ClusterConfiguration);
+                return new TestCluster(options);
+            }
+
+            public override void Dispose()
+            {
+                base.Dispose();
+                var dataManager = new AzureTableDataManager<TableEntity>(CheckpointerSettings.TableName, CheckpointerSettings.DataConnectionString);
+                dataManager.InitTableAsync().Wait();
+                dataManager.ClearTableAsync().Wait();
+            }
+
+            private static void AdjustClusterConfiguration(ClusterConfiguration config)
+            {
+                var settings = new Dictionary<string, string>();
+                // get initial settings from configs
+                ProviderSettings.WriteProperties(settings);
+                EventHubConfig.Value.WriteProperties(settings);
+                CheckpointerSettings.WriteProperties(settings);
+
+                // add queue balancer setting
+                settings.Add(PersistentStreamProviderConfig.QUEUE_BALANCER_TYPE, StreamQueueBalancerType.DynamicClusterConfigDeploymentBalancer.ToString());
+
+                // register stream provider
+                config.Globals.RegisterStreamProvider<EHStreamProviderWithCreatedCacheList>(StreamProviderName, settings);
+                config.Globals.RegisterStorageProvider<MemoryStorage>("PubSubStore");
+            }
+        }
+
+        public EHSlowConsumingTests(Fixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [Fact, TestCategory("EventHub"), TestCategory("Streaming"), TestCategory("Functional")]
+        public async Task EHSlowConsuming_ShouldFavorSlowConsumer()
```

Comment:
yes, configuration is already set up in VSO, it's just that currently none of the EH tests are marked as Functional, so they just don't run because of that. I'd start adding these tests using the Skippable approach, and then we can look at adding more of the other EH tests using the same approach